components/eventing-controller/pkg/handlers/jetstream_test.go

```diff
@@ -967,6 +967,10 @@ func TestSubscription_JetStreamServerRestart(t *testing.T) {
 	jsBackend := testEnvironment.jsBackend
 	defer testEnvironment.natsServer.Shutdown()
 	defer testEnvironment.jsClient.natsConn.Close()
+
+	// use File storage to restore streams
+	jsBackend.config.JSStreamStorageType = JetStreamStorageTypeFile
```

Comment:
(nit) lets have another test with `in-memory` mode?

Reply:
Yes fair point.

Reply:
Added two more tests.